src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml

```diff
@@ -81,6 +81,35 @@ tests:
                       65533,
                   ]
 
+    - label: "Step 4a: TH reads AttributeList from DUT"
+      PICS: "!PICS_EVENT_LIST_ENABLED && BINFO.S.Afffb"
```

Comment:
PICS `*.S.Afff*` should never be explicitly listed --> it is mandatory and cannot be omitted so no test should be conditional on it

Reply:
Also event_list is mandatory yet here we are, disabling it :/. Totally agree on the already "actually mandatory" bits though. Lets open a GH issue to clean those up (but outside of scope of this particular PR)